README.md

```diff
@@ -174,10 +171,15 @@ If connecting to a development server over http, make sure to turn on http suppo
 
 ### Run in the emulator
 
+Pick a version and execute the following:
```

Comment:
I don't know that "type of build" == "version"

```suggestion
Pick a type of build and execute the following:
```

You should also indicate how people can find the valid "type of build" supported.